builds/jenkins-build-pack-sls-app/Jenkinsfile

```diff
@@ -36,7 +36,7 @@ import groovy.transform.Field
 @Field def environment_logical_id
 
 node() {
-
+  try {
   sh 'rm -rf *'
```

Comment:
Now that we are removing the working directory, this line can be removed.

Reply:
Removed